site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -106,25 +106,30 @@ focus tracking are unchanged with this proposal.
 1. (Child opt-in) Group a set of focusable child elements under a single focusgroup.
 2. (Descendent opt-in) Focusable elements deeply nested can participate in a single focusgroup.
 3. (Wrap) Focusgroup can be configured to have wrap-around focus semantics.
-4. (Horizontal/vertical) A focusgroup can be configured to respond to either horizontal navigation
-   keys or vertical keys or both (to trivially reserve one axis of arrow key behavior for
+4. (Limit directional arrow keys) A focusgroup can be configured to respond to either inline (e.g., horizontal) navigation
```

Comment:
I recommend linking out to the CSSWG definitions of logical properties for inline and block. As eg: horizontal is the physical property of inline.

Reply:
Linked!